### PR TITLE
Strip special characters.

### PR DESCRIPTION
**Pull request by kalebhermes**

### CODE REVIEW

File: heroes-sidebar-master/freeRotation.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def scrapeRotation(self):
 		heroList = [Hero.text for Hero in preParsedFreeRotation]
 
 		for i in range(10):
-			heroList[i] = heroList[i].split("(", 1)[0].replace(' ','').replace('.','')
+			heroList[i] = heroList[i].split("(", 1)[0].replace(' ','').replace('.','').replce("'",'')
```
Comment (yuna9) [inline heroes-sidebar-master/freeRotation.py]: `replace`

**Review by yuna9 — CHANGES_REQUESTED** [review]

File: heroes-sidebar-master/freeRotation.py (hunk 1 above)
Comment (kalebhermes) [reply heroes-sidebar-master/freeRotation.py]: ;)

**Comment by kalebhermes** [comment]
Updated

**Review by yuna9 — CHANGES_REQUESTED** [review]
Can you rebase into one commit?

**Pull request closed without merging**